controllers/constants/caikit-metrics.json

```diff
@@ -0,0 +1,55 @@
+{
+    "metrics": {
+        "supported": "true",
+        "config": [
+            {
+                "title": "Number of requests",
+                "type": "REQUEST_COUNT",
+                "queries": [
+                    {
+                        "title": "Number of successful incoming requests",
+                        "query": "sum(increase(predict_rpc_count_total{namespace='${NAMESPACE}',code='OK',model_id='${MODEL_ID}',grpc_request='${TASK}'}[${RATE_INTERVAL}]))"
```

Comment:
what does the `${TASK}` variable need to be replaced with while making the final query? @christinaexyou

Reply:
The specific embeddings task, e.g. EmbeddingTaskRequest, SentenceSimilarityTaskRequest